### DISCUSSION TITLE
Ender 3 Printer

### DISCUSSION THREAD

**Discussion by john-helyar** [Jyers/Marlin#1740] [General Discussion]
Hi there 

I just wanted to know if it will work i have an Ender 3 Printer running a Creality 4.2.7 Silent Board will this Firmware work with it as since i added this board to my printer and upgraded the Firmware to the latest creality one it doesn't have the refresh option for the SD Card as i use a SD to Micro SD Adapter and its a pain switching off and on, i know you have a fix for it but wanted to ask before i tried

**Comment by Cal-LF** [comment]
@john-helyar Is this the E3V2 you changed the main board to V427? Jyers does not work with the old ender 3 screen.
Jyers FW will read the SD card by moving the main menu selection square from PRINT to another selection and back to PRINT.

**Reply by john-helyar** [reply]
Thank you for the reply
ok mine is the old Ender 3 i am looking to upgrade my screen to either the V2 version or the Touch screen version, for now i will just stick with switching off and on again

**Reply by Cal-LF** [reply]
@john-helyar The touchscreen is not supported by Jyers.

**Comment by mrrstrat** [comment]
I have the Jyers firmware on a 4.2.2 board E3 and a 4.2.7 on a Ender 3 Max with the screen updated to the E3V2 screen. Either board and that screen seem to work just fine.

**Comment by mrrstrat** [comment]
I noticed this as well - evidently the "clicked-in" state of the SD card is polled in some way and the firmware reports it as a card inserted but not able to be mounted. More of an annoying "fix" potential but when a mountable filesystem is detected the firmware happily mounts to it. This one I might dig into :-)
